### PR TITLE
Initial gitbook setup

### PR DESCRIPTION
**Pull request by tpluscode**
You can check the current use case documents published here: https://hydra-examples.netlify.com

The only real change was to split the intro document so that the first para becomes the main readme, as it is the default front page of gitbook.

I placed all other current documents unchanged in a drafts folder. I would like to massage them and move into a menu structure more reflecting areas of interest (to be discussed, I think of stuff like "operations", "collections", etc).

Also, in subsequent PRs, I'd like to think about individual pages and split the client-related bit from those purely relating to the descriptions and meaning. More to come

### CODE REVIEW

File: summary.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+* [Assumptions](assumptions.md)
+* Drafts
+  * [1 Entrypoint](api-documentation/1.entry-point.md)
+  * [1.1 Security considerations](1.1.security-considerations.md)
+  * [2 API Documentation](api-documentation/2.api-documentation.md)
+    * [2.1 Data structures](api-documentation/2.1.api-documentation-data-structures.md)
+    * [2.2 User document](api-documentation/2.2.api-documentation-user-document.md)
+    * [2.3 Complete API Documentation](api-documentation/2.3.complete-api-documentation.md/2)
```
Comment (angelo-v) [inline summary.md]: This link is not functional, I guess because of the "/2" at the end.

**Comment by angelo-v** [comment]
Really cool! Will it be moved to a "real" hydra domain?

**Comment by tpluscode** [comment]
I s'pose so. And I also happen to own `hydra.how` which we could use for a nice domain hack.

File: summary.md (hunk 1 above)
Comment (alien-mcl) [reply summary.md]: Yep - I've browsed several pages and links were not working in those places.

**Comment by alien-mcl** [comment]
Indeed - awesome! I wish we had those cases exposed 3 years ago.

**Comment by angelo-v** [comment]
Let's publish this :tada:

File: api-documentation/1.entry-point.md [hunk 2]
```diff
@@ -120,4 +120,4 @@ thus an explicit statement should appear so the client can easily discover all t
 
 ### Security considerations
 
-Please refer to [the "security considerations" document](1.1.security-considerations.md).
+Please refer to [the "security considerations" document](../1.1.security-considerations.md).
```
Comment (alien-mcl) [inline api-documentation/1.entry-point.md]:
```suggestion
Please refer to the [security considerations](../1.1.security-considerations.md)  document.
```

File: drafts/3.obtaining-events.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ It is possible now to display one event in the calendar.
 
 ### Extensions
 
-Please refer to (this document)[/3.1.extensions-considerations.md] for possible extension considerations.
+Please refer to [this document](./3.1.extensions-considerations.md) for possible extension considerations.
```
Comment (alien-mcl) [inline drafts/3.obtaining-events.md]:
```suggestion
Please refer to [this](./3.1.extensions-considerations.md) document for possible extension considerations.
```

File: drafts/9.adding-existing-resources-to-collections.md [hunk 4]
```diff
@@ -107,4 +107,4 @@ HTTP 200 OK
 In both cases the `schema:AddAction` operation expects an existing resource
 (with an `@id`) and just adds it to collection's `member` array. Unlike
 `schema:CreateAction` explained in
-[5.creating-new-event.md](https://github.com/HydraCG/Specifications/blob/master/drafts/use-cases/5.creating-new-event.md).
+['Creating a new event'](./5.creating-new-event.md).
```
Comment (alien-mcl) [inline drafts/9.adding-existing-resources-to-collections.md]:
```suggestion
[Creating a new event](./5.creating-new-event.md) document.
```

File: summary.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+* [Assumptions](assumptions.md)
+* Drafts
+  * [1 Entrypoint](api-documentation/1.entry-point.md)
+  * [1.1 Security considerations](1.1.security-considerations.md)
```
Comment (alien-mcl) [inline summary.md]:
```suggestion
    * [1.1 Security considerations](1.1.security-considerations.md)
```

**Comment by tpluscode** [comment]
What kind of outage?

**Comment by alien-mcl** [comment]
The site was down for about 15 mins. While I don't expect availability at 0.999 from a free service, I was just wondering how often does it happen.

**Pull request merged**